configuration/config-txt/boot.md

```diff
@@ -35,7 +35,13 @@ Set the `disable_commandline_tags` command to `1` to stop `start.elf` from filli
 
 ## kernel
 
-`kernel` is the alternative filename on the boot partition to use when loading the kernel. The default value on the Pi 1, Pi Zero, and Compute Module is `kernel.img`, and on the Pi 2, Pi 3, and Compute Module 3 it is `kernel7.img`. If `kernel8.img` is present on the Pi 3 or Compute Module 3, it will be loaded in preference and entered in 64-bit mode. **NOTE**: This must be an uncompressed kernel image file.
+`kernel` is the alternative filename on the boot partition to use when loading the kernel. The default value on the Pi 1, Pi Zero, and Compute Module is `kernel.img`, and on the Pi 2, Pi 3, and Compute Module 3 it is `kernel7.img`. On the Pi4, it is `kernel7l.img`. **NOTE**: This must be an uncompressed kernel image file.
```

Comment:
The "uncompressed kernel image" requirement applies to 64-bit kernels.

Reply:
But 32 bit kernels can be compressed or uncompressed? Is it always still called `kernel.img` regardless of whether it's compressed or not? (And is it worth mentioning what kernel compression schemes are supported?)
:man_shrugging: